lib/kubeclient/watch_stream.rb

```diff
@@ -8,6 +8,7 @@ def initialize(uri, http_options, formatter:)
         @uri = uri
         @http_client = nil
         @http_options = http_options
+        @http_options[:http_max_redirects] ||= Kubeclient::Client::DEFAULT_HTTP_MAX_REDIRECTS
```

Comment:
Referring to `Kubeclient::Client::DEFAULT_HTTP_MAX_REDIRECTS` feels slightly icky here, but this is the most direct way to get a consistent default

Reply:
I _think_ this could be omitted as WatchStream is normally passed what http_options returns.  I'll play with it later today.